docs/proposals/volume-unification.md

```diff
@@ -0,0 +1,65 @@
+## Abstract
+
+A proposal for enabling several Kubernetes resources to be available
```

Comment:
Needs more information.  I would say something like "A proposal for enabling users to create a single volume populated from secrets, configmaps, and downward API" instead.